src/Plug.vala

```diff
@@ -43,7 +43,7 @@ public class NotificationsPlug : Switchboard.Plug {
         Object (category: Category.PERSONAL,
                 code_name: "io.elementary.switchboard.notifications",
                 display_name: _("Notifications"),
-                description: _("Configure notification bubbles, sounds, and notification center"),
+                description: _("Adjust notification bubbles, sounds, and the notification center"),
```

Comment:
Oh this is a tough one. I'm not sure that "configure" or "adjust" sound completely right here. To me, "adjust" implies that there is some kind of granular control, but what we're really doing is very binary in terms of turning bubbles and sounds on or off and either allowing or disallowing them in the notification center. I'm not sure I have a great suggestion on how to improve this, but I'm not sure that this is a large enough improvement to invalidate existing translations